publish.sh

```diff
@@ -11,13 +11,13 @@ cd ${DISTRIBUTIONS[0]}/unpinned
 bzr init
 bzr add
 bzr commit -m "Packaging for ${VERSION}-0ubuntu1."
-bzr push --overwrite bzr+ssh://valhalla-routing@bazaar.launchpad.net/~valhalla-routing/+junk/valhalla_${VERSION}-0ubuntu1
+bzr push --overwrite bzr+ssh://${LAUNCHPADUSER}@bazaar.launchpad.net/~valhalla-core/+junk/valhalla_${VERSION}-0ubuntu1
```

Comment:
you will need to add to your environment `${LAUNCHPADUSER}` this will be the account used to publish updates to launchpad. if you want to keep using the shared account set it to `valhalla-routing` in your bashrc or whever you set your env